ibm/resource_ibm_schematics_job.go

```diff
@@ -0,0 +1,2040 @@
+/**
+ * (C) Copyright IBM Corp. 2021.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ibm
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/go-openapi/strfmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/IBM/go-sdk-core/v5/core"
+	"github.com/IBM/schematics-go-sdk/schematicsv1"
+)
+
+func resourceIBMSchematicsJob() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceIBMSchematicsJobCreate,
+		ReadContext:   resourceIBMSchematicsJobRead,
+		UpdateContext: resourceIBMSchematicsJobUpdate,
+		DeleteContext: resourceIBMSchematicsJobDelete,
+		Importer:      &schema.ResourceImporter{},
+
+		Schema: map[string]*schema.Schema{
+			"command_object": &schema.Schema{
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: InvokeValidator("ibm_schematics_job", "command_object"),
+				Description:  "Name of the Schematics automation resource.",
+			},
+			"command_object_id": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Job command object ID (`workspace-id, action-id or control-id`).",
+			},
+			"command_name": &schema.Schema{
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: InvokeValidator("ibm_schematics_job", "command_name"),
+				Description:  "Schematics job command name.",
+			},
+			"command_parameter": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Schematics job command parameter (`playbook-name, capsule-name or flow-name`).",
+			},
+			"command_options": &schema.Schema{
+				Type:        schema.TypeList,
+				Optional:    true,
+				Description: "Command line options for the command.",
+				Elem:        &schema.Schema{Type: schema.TypeString},
+			},
+			"job_inputs": &schema.Schema{
+				Type:        schema.TypeList,
+				Optional:    true,
+				Description: "Job inputs used by an action.",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"name": &schema.Schema{
```

Comment:
from these any of the one attribute is required ?

Reply:
Updated name and value fields as required. Again this is due to the generated code. Will check if there is a known issue with the code generator or raise an issue